modules/planning/tasks/deciders/speed_bounds_decider/st_boundary_mapper.cc

```diff
@@ -198,6 +199,14 @@ bool STBoundaryMapper::GetOverlapBoundaryPoints(
     return false;
   }
 
+  const auto* planning_status = PlanningContext::Instance()
+                                ->mutable_planning_status()
+                                ->mutable_change_lane();
+
+  double l_buffer = FLAGS_nonstatic_obstacle_nudge_l_buffer;
+  if (planning_status->status() == ChangeLaneStatus::IN_CHANGE_LANE) {
+    l_buffer = 0.0;
```

Comment:
0.0 is too aggressive.  e.g a vehicle in front of adc at the original lane may get ignored while status is IN_CHANGE_LANE?

Reply:
Set the value to 0.3m now. Thanks.